### ISSUE TITLE
What's missing to make it work for Flutter Web ? 

### ISSUE THREAD

**Issue by Tiska** [rebelappstudio/accessibility_tools#58]
Hi :) What's missing to make it work for Flutter Web ?

**Comment by aednlaxer** [comment]
Hi,

Looks like nothing is missing to make `accessibility_tools` available on web. However, there are couple of things that need to be changed:
1. pub.dev [automatically detects](https://dart.dev/tools/pub/pubspec#platforms) available platforms and for some reason (possibly because of number two below) it doesn't think web is supported. This could be fixed by listing all supported platforms in `pubspec.yaml`
2. There's [`!kIsWeb`](https://github.com/rebelappstudio/accessibility_tools/blob/729ba9c073b638551d1706970d45b249d0f73b27/lib/src/accessibility_tools.dart#L140) check in one place in the code that doesn't let overlay with errors to be shown. I can't recall why this condition is there but it seems to be there since the very first commit.

I've briefly tested the example app on web. Checkers, overlay and testing panel seem to work fine. There's one thing that I've found so far - scrolling gets glitchy when checkers overlay is on but it could be fixed separately.

I'll make these changes soon to make the package available on web.

**Comment by aednlaxer** [comment]
These changes have been made in #59 and released in [v2.2.3](https://pub.dev/packages/accessibility_tools/versions/2.2.3)

pub.dev now shows web as one of available platforms:
![image](https://github.com/user-attachments/assets/fba111f3-9808-48f6-833b-fcda68d42794)
